components/org.wso2.carbon.identity.local.auth.smsotp.provider/src/main/java/org/wso2/carbon/identity/local/auth/smsotp/provider/model/SMSData.java

```diff
@@ -29,72 +32,99 @@
 public class SMSData implements Serializable {
 
     private static final long serialVersionUID = 350777056982260622L;
+    private String body;
     private String toNumber;
-    private String fromNumber;
-    private String smsBody;
-    private SMSMetadata smsMetadata;
+    private String contentType;
+    private String httpMethod;
+    private Map<String, String> headers = new HashMap<>();
 
     /**
-     * Returns the from number.
-     * @return From number.
+     * Returns the body.
+     *
+     * @return Body.
      */
-    public String getFromNumber() {
-        return fromNumber;
+    public String getBody() {
+        return body;
     }
 
     /**
-     * Returns the SMS metadata {@link SMSMetadata}.
-     * @return SMS metadata.
+     * Returns the content type.
+     *
+     * @return Content type.
      */
-    public SMSMetadata getSmsMetadata() {
```

Comment:
Are these methods introduced recently ? Otherwise as we are changing the signature of the public methods there will be backward compatibility issues.

Reply:
Yes. These are introduced recently and not used in any other places.